lib/savings/view/savings_screen.dart

```diff
@@ -0,0 +1,100 @@
+// ignore_for_file: omit_local_variable_types, prefer_final_locals
+
+import 'package:flutter/material.dart';
+import 'package:fpb/assets/fpb_svg.dart';
+import 'package:fpb/l10n/l10n.dart';
+import 'package:fpb/savings/view/widgets/account_card.dart';
+import 'package:fpb/savings/view/widgets/account_type_tile.dart';
+import 'package:fpb/savings/view/widgets/savings_screen_title.dart';
+
+class SavingsPage extends StatefulWidget {
+  const SavingsPage({super.key});
+  static const String routeName = '/savings';
+
+  @override
+  State<SavingsPage> createState() => _SavingsPageState();
+}
+
+class _SavingsPageState extends State<SavingsPage> {
+  int pageIndex = 1;
+
+  void onChanged(int index) {
+    setState(() {
+      pageIndex = index;
```

Comment:
Why do you have this `pageIndex` variable and the `onChanged` method here?

Please make sure to always use a `StatefulWidget` only when necessary

Reply:
The pageIndex variable is in the onChanged method so that the changes can be reflected on the UI. This is due to the way the bottomnavbar is declared

Reply:
but I can't see where you use the method in this Widget